madgraph5amcatnlo.spec

```diff
@@ -1,14 +1,13 @@
-### RPM external madgraph5amcatnlo 2.6.7
-%define versiontag 2_6_7
-%define commit 307
+### RPM external madgraph5amcatnlo 2.7.3
+%define versiontag 2_7_3
 Provides: perl(Compress::Zlib)
 Provides: perl(List::Util)
-Source: http://cmsrep.cern.ch/cmssw/download/%{n}-%{realversion}.tgz
-#Source: https://launchpad.net/mg5amcnlo/2.0/2.5.x/+download/MG5_aMC_v%{realversion}.tar.gz
-#Source: https://launchpad.net/mg5amcnlo/2.0/2.6.x/+download/MG5_aMC_v%{realversion}.tar.gz
+Source: https://launchpad.net/mg5amcnlo/2.0/2.7.x/+download/MG5_aMC_v%{realversion}.py3.tar.gz
 Patch0: madgraph5amcatnlo-config
+# Compile and install internal and external packages
+#Patch1: madgraph5amcatnlo-compile
```

Comment:
@mrodozov , what is this patch and why it is not needed for cc8?

Reply:
followed your example on how to build it on cc8 from here:
https://github.com/cms-sw/cmsdist/commit/5b24f9e3ea4f7362e90dcc34c8b3a632a3da13e4#
the patch is a script that is used to compile madgraph

Reply:
do we need this patch for cc7? I thought the newer madgraph version did not need it that is why I had dropped it. So now if cc7 also has the newer version then do we need this patch?

Reply:
I'll check if it works without it and remove it if it isn't

Reply:
it's segfaulting. looks like this script is creating something that is needed for slc7. in short we need this patch